pkgs/development/compilers/scala/bare.nix

```diff
@@ -37,20 +37,17 @@ stdenv.mkDerivation (finalAttrs: {
   '';
 
   meta = with lib; {
-    description = "Research platform for new language concepts and compiler technologies for Scala";
-    longDescription = ''
-      Dotty is a platform to try out new language concepts and compiler technologies for Scala.
-      The focus is mainly on simplification. We remove extraneous syntax (e.g. no XML literals),
-      and try to boil down Scala’s types into a smaller set of more fundamental constructs.
-      The theory behind these constructs is researched in DOT, a calculus for dependent object types.
-    '';
-    homepage = "http://dotty.epfl.ch/";
+    description = "The Scala 3 compiler, also known as Dotty";
+    homepage = "https://scala-lang.org/";
     license = licenses.bsd3;
```

Comment:
@natsukagami We have [changed the license](https://github.com/scala/scala3/pull/8026) 4 years ago, can you update this field too please

cc/ @SethTisue